package.json

```diff
@@ -93,7 +93,7 @@
     "@types/faker": "^5.1.7",
     "@types/lodash": "^4.14.168",
     "@types/react-dom": "^18.0.3",
-    "eslint-plugin-jest-dom": "^4.0.1",
+    "eslint-plugin-jest-dom": "^5.1.0",
```

Comment:
`@testing-library/react` and earlier versions of `eslint-plugin-jest-dom` depended on different versions of `@testing-library/dom`, which caused different configurations for `@testing-library/dom` to be used and resulted in unexpected act warnings.

The 5.1.0 release of `eslint-plugin-jest-dom` made `@testing-library/dom` an optional dependency, so now only `@testing-library/react` depends on `@testing-library/dom`.